jni/logger_api.cpp

```diff
@@ -12,7 +12,7 @@ JNIEXPORT void JNICALL
 Java_org_openobservatory_measurement_1kit_jni_LoggerApi_setVerbose
   (JNIEnv *, jclass, jint verbose) {
     try {
-        mk::set_verbose(verbose);
+        mk::set_verbosity(verbose);
```

Comment:
Note to self: it would perhaps be optimal to have constant integers for the various log levels